bitvec.cabal

```
@@ -94,11 +93,12 @@ library
   else
     build-depends: ghc-bignum
 
-  if flag(libgmp)
-    extra-libraries: gmp
-    cpp-options: -DUseLibGmp
+  if flag(simd)
+    c-sources: cbits/bitvec_simd.c
+    cc-options: -O2 -fopenmp
```

Let's make `cabal check` happy.
```suggestion
    cc-options: -fopenmp
```